lib/swoosh/adapters/local.ex

```diff
@@ -22,6 +22,11 @@ defmodule Swoosh.Adapters.Local do
   @behaviour Swoosh.Adapter
 
   def deliver(%Swoosh.Email{} = email, _config) do
-    Swoosh.InMemoryMailbox.push(email)
+    id =
```

Comment:
Also, `Plug.Swoosh.MailboxPreview` needs more testing to catch that but I believe this change will break it.